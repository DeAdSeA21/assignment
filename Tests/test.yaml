dueDate: 2025-04-28
latePenalties: [10, 20, 100]
test:
  launch: ./loan
  timelimit: 10
  grading:
    - oracle: smart
      scoring: ByLine
      ws: true
      cap: 100
    - oracle: smart
      scoring: ByLine
      numbersOnly: true
      cap: 85
build:
  instructorFiles:
    exclude:
      - loan.cpp
  studentFiles:
    exclude:
      - makefile
  command: make -k
