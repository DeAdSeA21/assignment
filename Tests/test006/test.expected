Starting loan balance: Annual interest rate: Monthly payment: 
Month	Balance
1	996.00
2	992.00
3	988.00
4	984.00
5	980.00
6	976.00
7	972.00
8	968.00
9	964.00
10	960.00
11	956.00
12	952.00
13	948.00
14	944.00
15	940.00
16	936.00
17	932.00
18	928.00
19	924.00
20	920.00
21	916.00
22	912.00
23	908.00
24	904.00
25	900.00
26	896.00
27	892.00
28	888.00
29	884.00
30	880.00
31	876.00
32	872.00
33	868.00
34	864.00
35	860.00
36	856.00
37	852.00
38	848.00
39	844.00
40	840.00
41	836.00
42	832.00
43	828.00
44	824.00
45	820.00
46	816.00
47	812.00
48	808.00
49	804.00
50	800.00
51	796.00
52	792.00
53	788.00
54	784.00
55	780.00
56	776.00
57	772.00
58	768.00
59	764.00
60	760.00
